build/flatpak/metainfo.xml.in

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>${FLATPAK_ID}</id>
+  <name>${FLATPAK_APPNAME}</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>New Vector Ltd</developer_name>
+  <summary>Create, share, communicate, chat and call securely, and bridge to other apps</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://element.io/</url>
+  <url type="help">https://element.io/help</url>
+  <url type="translate">https://translate.element.io/</url>
+  <description>
+    <p>More than group chat: communication</p>
+    <ul>
+      <li>Communicate with your team and out of network colleagues more efficiently: use dedicated rooms which persist information from their creation and forever.</li>
+      <li>Forget group emails: join or create rooms per topic, per team, per event… Decide the level of transparency you want to provide across the organisation or project.</li>
+      <li>Cut through the noise by creating notifications that are customised by you and for you.</li>
+      <li>Grab the attention of your colleague by calling out their name and don’t miss a thing with keyword alerts.</li>
+      <li>Deploy bots for fun or practical use with our integrations store.</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot>
+      <caption>Element logged-in frontend</caption>
+      <image type="source">https://user-images.githubusercontent.com/2221064/137519256-655fe0f9-55f0-44a4-b681-83711654fff0.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="${FLATPAK_VERSION}" date="${FLATPAK_DATE}"/>
+  </releases>
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">intense</content_attribute>
+    <content_attribute id="social-audio">intense</content_attribute>
+    <content_attribute id="social-contacts">intense</content_attribute>
+  </content_rating>
+  <provides>
+    <id>im.riot.Riot</id>
```

Comment:
Are we stuck with this from historical precedent?

Reply:
Yes, kind of, keep in mind that this entry basically says "I provide the same capabilities as the app with the identifier `im.riot.Riot`" it's not the identifier of this app.